src/apps/explorer/components/SummaryCardsWidget/SummaryCards.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useEffect, useState } from 'react'
+import styled, { css } from 'styled-components'
+import { media } from 'theme/styles/media'
+import { formatDistanceToNowStrict } from 'date-fns'
+
+import {
+  getMatchingScreenSize,
+  subscribeToScreenSizeChange,
+  MediumDownQueries,
+  TypeMediaQueries,
+} from 'utils/mediaQueries'
+import { Card, CardContent } from 'components/common/Card'
+import { CardRow } from 'components/common/CardRow'
+import { TotalSummaryResponse } from '.'
+import { abbreviateString } from 'utils'
+
+const BatchInfoHeight = '19.6rem'
+const DESKTOP_TEXT_SIZE = 1.8 // rem
+const MOBILE_TEXT_SIZE = 1.65 // rem
+
+const WrapperCardRow = styled(CardRow)`
+  max-width: 70%;
+
+  ${media.mobile} {
+    max-width: 100%;
+  }
+`
+
+const DoubleContentSize = css`
+  min-height: ${BatchInfoHeight};
+`
+const WrapperColumn = styled.div`
+  /* Equivalent to use lg={8} MUI grid system */
+  flex-grow: 0;
+  max-width: 66.666667%;
+  flex-basis: 66.666667%;
+  padding-right: 2rem;
+
+  > div {
+    margin: 1rem;
+    max-height: ${BatchInfoHeight};
+  }
+
+  ${media.mediumDownMd} {
+    flex-grow: 0;
+    max-width: 100%;
+    flex-basis: 100%;
+  }
+`
+const DoubleCardStyle = css`
+  ${DoubleContentSize}
+
+  ${media.mediumDownMd} {
+    min-height: 16rem;
+  }
+`
+const WrappedDoubleCard = styled(Card)`
+  ${DoubleCardStyle}
+`
+
+const CardTransactions = styled(Card)`
+  ${media.mediumDownMd} {
+    ${DoubleCardStyle}
+  }
+`
+
+const WrapperDoubleContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+
+  ${media.mediumDownMd} {
+    gap: 2rem;
+  }
+`
+
+interface SummaryCardsProps {
+  summaryData: TotalSummaryResponse | undefined
+  children: React.ReactNode
+}
+
+function calcDiff(a: number, b: number): number {
+  return (a - b === 0 ? 0 : (100 * (a - b)) / b) || 0
+}
+
+function getColorBySign(n: number): string {
+  const sign = Math.sign(n)
+
+  if (sign === 1) {
+    return 'green'
+  } else if (sign === -1) {
+    return 'red1'
+  }
+
+  return 'grey'
+}
+
+export function SummaryCards({ summaryData, children }: SummaryCardsProps): JSX.Element {
+  const { batchInfo, dailyTransactions, totalTokens, dailyFees, isLoading } = summaryData || {}
+  const [resolution, setResolution] = useState<TypeMediaQueries>(getMatchingScreenSize())
+  const isMediumAndBelowResolution = MediumDownQueries.includes(resolution)
+  const isDesktop = !isMediumAndBelowResolution
+  const valueTextSize = isDesktop ? DESKTOP_TEXT_SIZE : MOBILE_TEXT_SIZE
+  const rowsByCard = isDesktop ? '2row' : '3row'
+  const diffTransactions = (dailyTransactions && calcDiff(dailyTransactions.now, dailyTransactions.before)) || 0
+  const diffFees = (dailyFees && calcDiff(dailyFees.now, dailyFees.before)) || 0
+
+  useEffect(() => {
+    const mediaQuery = subscribeToScreenSizeChange(() => setResolution(getMatchingScreenSize()))
+    return (): void => mediaQuery()
+  }, [])
```

Comment:
we could move this to its own hook that returns the screen size and then maybe have another utility hook based on the previous one with something like:

```js
const isDesktop = useMediaBreakpoint(['xl', 'lg'])
```

Reply:
I like the idea!